compose-builder/add-security.yml

```diff
@@ -68,8 +68,8 @@ services:
   secretstore-setup:
     image: ${CORE_EDGEX_REPOSITORY}/security-secretstore-setup${ARCH}:${CORE_EDGEX_VERSION}${DEV}
     user: "root:root" # must run as root
-    container_name: edgex-secretstore-setup
-    hostname: edgex-secretstore-setup
+    container_name: edgex-security-secretstore-setup
+    hostname: edgex-security-secretstore-setup
```

Comment:
are we going back to edgex-?  i remembered it was like that before

Reply:
We never change away for `edgex-` for container/hostname. This just change it to use the service key appropriately

Reply:
ok got it.